command_handlers/link_preview.py

```
@@ -111,9 +115,42 @@ async def _cmd_link_preview_impl(self, event: AstrMessageEvent):
+                                int(raw["message_id"]),
+                                int(raw["time"]),
+                                int(raw["self_id"]),
+                            ),
+                        ):
+                            logger.debug(
+                                "[NitterTweets] link preview arbiter lost, "
+                                "skipping: umo=%s",
+                                umo,
+                            )
+                            return
+                    except Exception as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)